### PR TITLE
Early bird

### PR DESCRIPTION
**Pull request by lmaCode**
Fixes # .

Make sure these boxes are checked before creating your pull request -- thank you!

- [x] Add Blog Header, including title, author, date.

e.g.

``` md
---
 title: "'Score Your Face Photo' a ML&Wechaty practice"
 date: 2017-09-18 09:00 +0800
 author: huyingxi
 ---
 > Author: [@huyingxi](https://github.com/huyingxi/wechaty_selfie) enjoying ML&Wechaty at BUPT
```

- [x] Keep all filenames & url as lowercase, and use `-` to connect words instead of space. e.g. `2017-10-06-wechat-pc-impactor` instead of `2017-10-06-WeChat PC Impactor`,
- [x] Embed the photo & video before publishing, save all external file to the blog `/download/2018` directory.
- [x] Add `<!--more-->` section to just show the abstract in blog homepage.
- [x] Add teaser image for the blog
- [x] Add your self as an author to the author data file at <https://github.com/wechaty/wechaty.github.io/blob/master/_data/authors.yml> with your GitHub username. And please also remember to reference yourself with the Github username in your blog post, so that we can have nice author information besides your blog after we published it!

[See more](https://github.com/wechaty/wechaty.github.io#contribute-guideline)

### CODE REVIEW

**Comment by lmaCode** [comment]
Who can tell me what caused NPM build Test failure?

**Comment by csjuXYZ** [comment]
> Who can tell me what caused NPM build Test failure?

MD文档有多余的空行或者空格,具体可以在details看出项错误的行数

**Comment by lijiarui** [comment]
Please make CI green then I can review your PR.

**Comment by lmaCode** [comment]
> Please make CI green then I can review your PR.

Done. Thank you all. @csjuXYZ @lijiarui

File: _posts/2020-07-23-early-bird-chatbot.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+3. 一个你能长期获得帮助的社群，从找实习、全职到职业发展等
+
+2020的秋招马上开始了，早鸟计划扩展出刷题、项目、简历、求职、内推、面试和实习等十余个分队，每个分队涉及众多国内外名校的学生，将分别以不同的主题建群。为了更方便地维护求职信息群，急需生成一款社群管理工具。受限于部分用户无法登录`web 协议` 的wechaty。所以使用基于`iPad 协议`的wechaty-puppet-padplus来实现功能。
+
+> Wechaty 是一个开源的的个人号微信机器人接口，是一个使用 Typescript 构建的 Node.js 应用。支持多种微信接入方案，包括网页，ipad，ios，windows， android 等。同时支持 Linux, Windows, Darwin(OSX/Mac) 和 Docker 多个平台。
+
+## 如何获取免费 Token
+
+官方文档中提供了免费 token 的获取方式。
+[https://github.com/juzibot/Welcome/wiki/Support-Developers](https://github.com/juzibot/Welcome/wiki/Support-Developers)
```
Comment (lijiarui) [inline _posts/2020-07-23-early-bird-chatbot.md]: change  to： https://github.com/juzibot/Welcome/wiki/Everything-about-Wechaty

**Review by lijiarui — CHANGES_REQUESTED** [review]

File: _posts/2020-07-23-early-bird-chatbot.md (hunk 1 above)
Comment (lmaCode) [reply _posts/2020-07-23-early-bird-chatbot.md]: Done. Please check it. @lijiarui

**Review by lijiarui — APPROVED** [review]
Good job

**Comment by lijiarui** [comment]
Congratulations! 🎉 🎉 🎉

You become a wechaty contributor successfully! 👏 👏 👏

Please:

- [ ]  Add my wechat: ruirui_0914 and send me this pr link. I will invite you into Wechaty Contributor Room
- [ ] Update your profile here: [Contributor List](https://docs.google.com/spreadsheets/d/1XcDoIczyIclqXP1p90Sz7S0n4Q22xjFzJmjPFlU2g1E/edit#gid=0), then other contributors will know you better!
- [ ]  Join wechaty org by clicking https://github.com/wechaty

image

**Pull request merged**